spec/helpers/option.js

```diff
@@ -21,22 +21,23 @@ describe('option helper', function () {
         ], done);
     });
 
-    it('does not access prototype props', function (done) {
-        context.options.__proto__ = {x: 'yz'};
-        runTestCases([
-            {
-                input: `{{option "x"}}`,
-                output: ``,
-            },
-        ], done);
-    });
+    // uncomment when 3rd-party version is replaced
+    // it('does not access prototype props', function (done) {
+    //     context.options.__proto__ = {x: 'yz'};
+    //     runTestCases([
+    //         {
+    //             input: `{{option "x"}}`,
+    //             output: ``,
+    //         },
+    //     ], done);
+    // });
 
-    it('accepts SafeString paths', (done) => {
-        runTestCases([
-            {
-                input: `{{option (concat 'a.b.' 'c')}}`,
-                output: `d`,
-            },
-        ], done);
-    });
+    // it('accepts SafeString paths', (done) => {
```

Comment:
Shouldn't it be supported now?

Reply:
The old `option` helper (and also `get`) only works with SafeString for top-level properties, not nested properties, so this test will fail on the old version.